webknossos/webknossos/skeleton/node.py

```diff
@@ -8,10 +8,10 @@
 Vector3 = Tuple[float, float, float]
 
 
-@attr.define()
+@attr.define(order=True)
```

Comment:
This uses all (non-excluded) attributes for the order. That's also why I excluded _skeleton, since this might introduce infinite recursive loops if two nodes are on the same position. I'll add a comment :+1: